### ISSUE TITLE
Percentage symbol (%) showing when starting the terminal

### ISSUE THREAD

**Issue by truvaking** [zeit/hyper#4084]
- [X] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [X] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: macOS Catalina 10.15.2
- **Hyper.app version**: 3.0.2
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**: It's the default one
- **Relevant information from devtools** : N/A
- **The issue is reproducible in vanilla Hyper.app**: Vanilla

## Issue

Basically there's this `%`symbol showing up when I start the terminal. How do I remove it? 

<img width="1641" alt="Screenshot 2019-12-19 at 16 27 22" src="https://user-images.githubusercontent.com/26815719/71186048-d3079500-227c-11ea-87a5-ebeb2d7e795f.png">

**Comment by LabhanshAgrawal** [comment]
https://github.com/zeit/hyper/issues/3586#issuecomment-541983255

**Comment by truvaking** [comment]
Thanks, couldn't find any issue when searched for the keyword '%'. Sorry for bothering..
